AutonomousDemo/VehicleMotion.cs

```diff
@@ -26,18 +27,29 @@ public Velocity CalculateCurrentVelocity(double meters, int seconds)
 
         public Velocity DecelerateVelocityPerSecond(Velocity velocity)
         {
-            var decelerationRate = -3.4; //m/s^2
             //less than 3.4?
             if(velocity.Speed == 0)
             {
                 return velocity;
             }
-            var decreasedSpeed = velocity.Speed + decelerationRate;
+            var decreasedSpeed = velocity.Speed - Rate;
             velocity.Speed = decreasedSpeed;
             //something about direction
 
             return velocity;
         }
+        public Velocity AccelerateVelocityPerSecond(Velocity velocity)
```

Comment:
There's something redundant in this method name, in that velocity is meters per second and acceleration is meters per second per second... so I dunno what AccelerateVelocityPerSecond means.